arcgis-ios-sdk-samples/Features/Add delete related features/RelatedFeaturesViewController.swift

```diff
@@ -56,7 +56,7 @@ class RelatedFeaturesViewController: UITableViewController {
         parameters.orderByFields = [AGSOrderBy(fieldName: "OBJECTID", sortOrder: .descending)]
         
         // query for species related to the selected park
-        self.originFeatureTable.queryRelatedFeatures(for: self.originFeature, parameters: parameters) { [weak self] (results: [AGSRelatedFeatureQueryResult]?, error: Error?) in
+        self.originFeatureTable.queryRelatedFeatures(for: self.originFeature, parameters: parameters, queryFeatureFields: .loadAll) { [weak self] (results: [AGSRelatedFeatureQueryResult]?, error: Error?) in
```

Comment:
When I was testing with the dev feature table, it doesn't automatically populate all the fields from the table in a query. Even if the original code works fine with our current data source, I decide to add this parameter to be more defensive.